cmake/modules/SearchInstalledSoftware.cmake

```diff
@@ -231,6 +231,39 @@ if(builtin_lzma)
 endif()
 
 
+#---Check for LZ4--------------------------------------------------------------------
+if(NOT builtin_lz4)
```

Comment:
Not easily: the true-branch of the first `if`-statement can affect the outcome of the second conditional.